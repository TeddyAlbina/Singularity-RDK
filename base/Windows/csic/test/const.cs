using System;
class D {
	public const float B=((int)A.X)*3.0F;
	public const int C=3;
	public const A E=A.Y;
	public static void Main() {
		Console.WriteLine("B={0}", B);
		Console.WriteLine("C={0}", C);
		Console.WriteLine("D={0}={1}", E, (int)E);
	}
}
enum A { X=D.C, Y }
