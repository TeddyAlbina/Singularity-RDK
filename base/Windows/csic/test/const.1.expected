B=9
C=3
D=Y=4
